src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,68 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+    private static final DateTimeFormatter DATE_FORMATTER =
+            DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
+    public String getSalaryInfo(String[] names, String[] data,
```

Comment:
Try to divide your logic into private methods and simplify your solution. The method should contain 15-20 lines.